### ISSUE TITLE
Add parameter for a test project location

### ISSUE THREAD

**Issue by robs** [projectkudu/KuduScript#37]
Hi Guys,

Given that this _can_ be used as a sort of lightweight CI service do you have any plans to add a parameter for a test project location to the template builder?

I mention it as I haven't yet used a generated script that I haven't added a test that halts deployment.

Awesome project by the way!

**Comment by davidebbo** [comment]
It's one of those things that have come up but that we've never done. We'd happily take a PR if you feel up to the challenge!

**Comment by robs** [comment]
Heh, well, I've only looked at adding them for .net projects (vs console and xunit for dnx). 
I could work it out for those but the node and basic ones, no idea.
I guess with this sort of project if you cater for one you have to cater for all.

I'll have a look but I've never done anything with node aside from getting the package for this so it may take a while. In the mean time, if anyone beats me to it I won't be offended :)

**Comment by davidebbo** [comment]
Not necessarily. I think it would be fine to have this be .NET only to start with. In one of our samples, we use [this logic](https://github.com/KuduApps/MvcAppWithTests/blob/7b7f17d5636b1a76c1e67bef5c4edcb1818b3e08/deploy.cmd#L82-L85) to run tests.

**Comment by robs** [comment]
Ok, makes sense, so the PR should have logic to throw if the param is included for a non .net project?

**Comment by davidebbo** [comment]
Yes, would be good to fail and just say it's not yet supported.
